gensrc/thrift/FrontendService.thrift

```diff
@@ -477,6 +477,12 @@ struct TListMaterializedViewStatusResult {
     1: optional list<TMaterializedViewStatus> materialized_views
 }
 
+// Pagination cursor for request segmentation
+struct TRequestPagination {
+    1: optional i64 offset
```

Comment:
offset is not used?

Reply:
yes, not used, but reserved for future usage

Reply:
in theory we can use it to implement the pagination